src/services/schema-export/tests/models/all/kotlin/ChildEmbeddedType.kt

```diff
@@ -0,0 +1,10 @@
+// Please note : @LinkingObjects and default values are not represented in the schema and thus will not be part of the generated models
+package your.package.name.here
+
+import io.realm.RealmObject
+
+open class ChildEmbeddedType : RealmObject() {
```

Comment:
This should be 
```
@RealmClass(embedded = true)
open class ChildEmbeddedType : RealmObject() {
```

and `io.realm.annotations.RealmClass` should be added to the list of imports.